Tools/i18n/makelocalealias.py

```diff
@@ -100,16 +101,15 @@ def parse_glibc_supported(filename):
 def pprint(data):
     items = sorted(data.items())
     for k, v in items:
-        print('    %-40s%a,' % ('%a:' % k, v))
+        print(f"    {k!a:<40}{v!a},")
```

Comment:
Do we still need to ensure this is printed as ASCII? I would imagine not.
```suggestion
        print(f"    {k:<40}{v},")
```

Reply:
It's better to enforce ASCII here, since encoding names need to be ASCII.